src/utils/chart-utils.tsx

```diff
@@ -137,25 +137,25 @@ export const renderCustomizedTooltipBarChart = ({
       >
         {name}
       </Typography>
-      {billableProject &&
+      {billableProject !== undefined && billableProject !==0 &&  
```

Comment:
I don't think these changes to any of the conditions in this file are necessary?

Reply:
I think we need these changes. 0 displays in tooltip, when value do not exist (it returned the first part of the falsy expression).